tests/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseToSpecificMethodRector/Wrong/wrong.php.inc

```diff
@@ -5,7 +5,7 @@ final class MyTest extends \PHPUnit\Framework\TestCase
     public function test()
     {
         $this->assertTrue(array_key_exists('...', ['...']), 'argument');
-        $this->assertTrue(in_array('...', ['...']), 'argument');
+        $this->assertTrue(in_array('...', ['...'], true), 'argument');
```

Comment:
Could you also add 

```php
$this->assertTrue(in_array('...', ['...']), 'argument');
```

so we know this case is still covered?

Reply:
I'll revert the `array_search`, so we know that both cases are covered, without expand our tests :)